lib/core/include/irods_server_properties.hpp

```diff
@@ -56,6 +57,10 @@ namespace irods
         /// @brief capture server_config.json
         void capture_json(const std::string& _filename);
 
+        bool exists(const std::string_view _key) const {
```

Comment:
@alanking @trel I can see this being renamed to `contains`. What do you think?

Reply:
if we're doing full name comparison...  i think `exists` is fine/better.

Reply:
I changed it to `contains` before you responded. Figured it made sense because the `key_value_proxy` uses `contains` instead of `exists`.